lisa/tools/lsmod.py

```diff
@@ -48,3 +50,44 @@ def module_exists(
             return True
 
         return False
+
+    def get_used_by_modules(
+        self,
+        mod_name: str,
+        no_info_log: bool = True,
+        no_error_log: bool = True,
+        no_debug_log: bool = True,
+    ) -> List[str]:
+        """
+        Returns a list of modules that are using the given module name.
+        """
+        result = self.run(
+            sudo=True,
+            force_run=True,
+            no_info_log=no_info_log,
+            no_error_log=no_error_log,
+            no_debug_log=no_debug_log,
+            expected_exit_code=0,
+        )
+        used_by_modules = []
+        for match in self.__output_pattern.finditer(result.stdout):
```

Comment:
Use LISA built-in regexp wrappers, it makes code simpler.

Reply:
updated